presto-main/src/main/java/com/facebook/presto/operator/Driver.java

```diff
@@ -306,12 +307,19 @@ private void processNewSources()
         }
 
         Optional<ListenableFuture<?>> result = tryWithLock(100, TimeUnit.MILLISECONDS, () -> {
-            ListenableFuture<?> future = processInternal();
+            ListenableFuture<?> future = processInternal(createTimer());
             return updateDriverBlockedFuture(future);
         });
         return result.orElse(NOT_BLOCKED);
     }
 
+    private OperationTimer createTimer()
+    {
+        return new OperationTimer(
+                driverContext.isCpuTimerEnabled(),
+                driverContext.isCpuTimerEnabled() && driverContext.isVerboseStats());
```

Comment:
What do you guys think about renaming the `verboseStats`-related methods/configs (`task.verbose-stats`)? This flag now enables operation CPU time tracking only and nothing more. However, its name sounds more than that.

Reply:
That's a good idea. I'm going to change the name to `isOperatorCpuTimerEnabled()`

Reply:
Actually `isPerOperatorCpuTimerEnabled`